modules/get-started/pages/intro-to-rpk.adoc

```diff
@@ -20,28 +20,21 @@ ifndef::env-cloud[]
 * xref:get-started:rpk-quickstart.adoc[]
 endif::[]
 
-== Specify command properties
+== Specify configuration properties
 
 You can specify `rpk` command properties in the following ways:
 
+* Create an xref:get-started:config-rpk-profile.adoc[]
 * Specify the appropriate flag on the command line.
 * Define the corresponding environment variables.
 +
 Environment variable settings only last for the duration of a shell session.
 
```

Comment:
Removing references to the `rpk` section as we want to move everything we can to profiles and the `rpk` section of the redpanda.yaml is deprecated.

It will always be maintained, but any new feature will be added to profiles and not there.

w.r.t modifying the `rpk.yaml`, we prefer the user to use `rpk profile` commands to do so and not manually edit the `rpk.yaml`

Reply:
@r-vasquez Should this line in the next paragraph also be changed then to remove the part about `redpanda.yaml`?
_If the environment variable is not set, the value in the `rpk.yaml` configuration file is used, if that file is available, otherwise the value in the `redpanda.yaml` configuration file is used._

Reply:
@r-vasquez should we also remove the `rpk` bullet from [this page](https://docs.redpanda.com/current/manage/cluster-maintenance/node-property-configuration/)?

Reply:
I think it's fine to leave both as:

> If the environment variable is not set, the value in the rpk.yaml configuration file is used, if that file is available, otherwise the value in the redpanda.yaml configuration file is used.

It's useful information on how rpk parses the properties but doesn't encourage users to use the redpanda.yaml

And from the link, it's fine to keep it.